catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/transformer/CswRecordMapperFilterVisitor.java

```diff
@@ -577,6 +578,9 @@ private double getDistanceInMeters(double distance, String units) {
     return new Distance(distance, linearUnit).getAs(LinearUnit.METER);
   }
 
+  @SuppressWarnings(
+      "squid:S2177" /* This method doesn't and isn't supposed to override it's parent;
+                    it s a copy of a parent function that wasn't inherited */)
```

Comment:
nit :  `This method doesn't and isn't supposed to override its parent; it's a copy of a parent function that wasn't inherited`